azure-pipelines.yml

```diff
@@ -4,7 +4,7 @@ trigger:
 
 variables:
   buildConfiguration: 'Release'
-  cakeVersion: 0.34.1
+  cakeVersion: 0.38.0
```

Comment:
This should be 0.38.5, as this is the Cake version we're building with. (Not the Cake version we're building against...)

Reply:
I've pushed this change (along with the rebase).  Thanks!